source/Account.cpp

```diff
@@ -211,19 +213,22 @@ string Account::Step(int64_t assets, int64_t salaries)
 	
 	// If you made payments of all three types, the punctuation needs to
 	// include commas, so just handle that separately here.
+	string salariesPaidDisplay = Format::Number(salariesPaid),
```

Comment:
Rather than define new variables you can simply wrap the numbers when they are being printed. While not expensive to do, this means only the numbers that need to be formatted (e.g. non-zero numbers) will be formatted.

e.g.
`out << Format::Number(salariesPaid) << ....`